new/analyse/add-messages-to-packages.ts

```diff
@@ -0,0 +1,35 @@
+import type { Module } from '../types';
+
+import { logger } from '../helpers/logger';
+import { clone } from 'ramda';
+
+const { error, log } = logger('[analyse] add messages to packages');
+
+export const addMessagesToPackages: Module = (env) => {
+  if (!env.commitMessages) {
+    const errorMessage = 'missing "commitMessages" on env object';
```

Comment:
Reminds me of an overall testcase (guess it makes already sense to collect them somewhere):
If there are no relevant commit messages (e.g. only chore), the script does nothing (except logging a message) and exits with `0`.